front-end/src/components/main-page/MainPage.jsx

```diff
@@ -69,14 +70,16 @@ class MainPage extends Component {
           </div>
           
         </div>
-          <div className="justify-center flex flex-col gap-y-3">
-            <div className="bg-off-white h-[75vh] box-border drop-shadow-md rounded-xl p-5 text-lg font-cabin mb-5">
-              JSON here
+          <div className="w-1/4 h-7/8 justify-center flex flex-col gap-y-3">
+            <div className="bg-off-white h-[75vh] box-border drop-shadow-md rounded-xl p-5 text-lg font-cabin mb-5 overflow-y-auto">
+                <TranscriptDescription></TranscriptDescription>
```

Comment:
I think we can just do this:
```suggestion
                <TranscriptDescription />
```